core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3052,7 +3052,7 @@ public void run(Reactor session) throws Exception {
                 // initialize views by inserting the default view if necessary
                 // this is both for clean Jenkins and for backward compatibility.
                 if(views.size()==0 || primaryView==null) {
-                    View v = new AllView(Messages.Hudson_ViewName());
+                    View v = new AllView("all");
```

Comment:
This looks like a reasonable default for new All views. No need to edit existing ones. Would just cause confusion and broken links.